src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -558,6 +558,7 @@ static bool _isValidImageMimeTypeAndEncoding(const char** href, const char** mim
 static unique_ptr<Picture> _imageBuildHelper(SvgLoaderData& loaderData, SvgNode* node, const Box& vBox, const string& svgPath)
 {
     if (!node->node.image.href) return nullptr;
+    if (strlen(node->node.image.href) == 0) return nullptr;
```

Comment:
I'd merge these two together:
```
if (!node->node.image.href || !strlen(node->node.image.href)) return nullptr;
```
but up to you

Reply:
Thanks! I updated pr